### ISSUE TITLE
Add test firmware fido2 cert hashes

### ISSUE THREAD

**Issue by szszszsz** [Nitrokey/pynitrokey#266]
Add test firmware fido2 cert hashes:
- `f1ed1aba24b16e8e3fabcda72b10cbfa54488d3b778bda552162d60c6dd7b4fa`

**Comment by szszszsz** [comment]
On a second thought, probably this should make some warning about using the test certificate

**Comment by robin-nitrokey** [comment]
I agree that this should cause a warning (or maybe just a different error message?).  This command is mainly intended for end users, and a test certificate should not be accepted in that setting.

**Comment by szszszsz** [comment]
Agreed. Let's correct that as you suggest

**Comment by robin-nitrokey** [comment]
We could also add a flag to `nk3 test` that can be used for dev devices.  Currently, this would mean adding the test FIDO2 cert to the list of accepted certs.

**Comment by jans23** [comment]
Test certificates should fail with the normal `test` command but could pass with an additional parameter such as `test --development` for example.

**Comment by robin-nitrokey** [comment]
Yes, that’s what I had in mind.

**Comment by robin-nitrokey** [comment]
This commit shows how this option could be added: https://github.com/Nitrokey/pynitrokey/pull/269/commits/e32c844995b7a03d99c66674d86fc8cb1d81e85d
